internal/types/webhook.go

```diff
@@ -18,11 +18,13 @@ type WebhookEvent struct {
 
 // Common webhook event names
 const (
-	WebhookEventInvoiceCreateDraft         = "invoice.create.drafted"
-	WebhookEventInvoiceUpdateFinalized     = "invoice.update.finalized"
-	WebhookEventInvoiceUpdatePayment       = "invoice.updated.payment"
-	WebhookEventInvoiceUpdateVoided        = "invoice.update.voided"
-	WebhookEventInvoiceAmountsRecalculated = "invoice.amounts.recalculated"
+	WebhookEventInvoiceCreateDraft          = "invoice.create.drafted"
+	WebhookEventInvoiceUpdateFinalized      = "invoice.update.finalized"
+	WebhookEventInvoiceUpdatePayment        = "invoice.updated.payment"
+	WebhookEventInvoiceUpdateVoided         = "invoice.update.voided"
+	WebhookEventInvoiceAmountsRecalculated  = "invoice.amounts.recalculated"
+	WebhookEventInvoicePaymentOverdue       = "invoice.payment.overdue"
+	WebhookEventInvoicePaymentStatusChanged = "invoice.payment.status.changed"
```

Comment:
same way maybe we can skip this and only fire `"invoice.update.payment"` as something related to payment has changed